src/verify.ts

```diff
@@ -54,6 +60,31 @@ export const verifyCertificate = (certificate: pki.Certificate | string, caPem?:
     };
 };
 
+export function verifyCertificateNotRevoked(signaturePem: string): Promise<{revocationError?: string, isRevoked: boolean}> {
+    const prepped = crypto.createPublicKey({key: CERTIFICATION_SERVER_PUBLIC_KEY});
+    const rand = `${Math.random() * 1000}`;
+    const verify = crypto.createVerify('RSA-SHA512');
+
+    verify.update(rand);
+
+    return fetch(CERTIFICATION_SERVER_URL, {
```

Comment:
Will this work in desktop when the user wants to use a proxy?
I'm genuinely asking, I don't know if we should allow the caller to inject their own `fetch` to make it work or if it already works.

Reply:
I've added an option to perform revocation check or not when verifying signatures